apps/emloyees-management/src/app/home/home.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from "@angular/core";
+import { Router } from '@angular/router';
+import { AuthService } from '@dashasorg/auth';
+
+@Component({
+  selector: 'dashasorg-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent {
+  constructor(private authService: AuthService, private router: Router) {}
+
+  logout() {
```

Comment:
will be better to create some navigation panel with the logout button

Reply:
next iteration